monkey/monkey_island/cc/ui/src/components/configuration-components/ValidationFormats.js

```diff
@@ -1,13 +1,22 @@
 const ipRegex = '((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)'
 const cidrNotationRegex = '([0-9]|1[0-9]|2[0-9]|3[0-2])'
 const hostnameRegex = '^([A-Za-z0-9]*[A-Za-z]+[A-Za-z0-9]*.?)*([A-Za-z0-9]*[A-Za-z]+[A-Za-z0-9]*)$'
+// path starts with `/` OR `$`
+const linuxDirRegex = '^/|\\$'
+// path starts like `C:\` OR `C:/` OR `$` OR `%abc%`
```

Comment:
How is a path starting with $ a valid absolute path? What does the $ indicate?

Reply:
Environment variables start with `$`.

Reply:
Can you link docs? Tested? 
![image](https://user-images.githubusercontent.com/36815064/124267372-d56ccf80-db40-11eb-9f0b-cb49d759427f.png)